test/trip_dispatch_test.rb

```diff
@@ -121,5 +122,151 @@ def build_test_dispatcher
         end
       end
     end
+    
+    describe "Find available Driver" do
```

Comment:
I had trouble finding the tests that checked on the functionality of .request_trip. You may consider calling attention to the fact you are testing request_trip in the `describe` string.